src/main/java/in/siva/util/ConnectionUtil.java

```diff
@@ -0,0 +1,83 @@
+package in.siva.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class ConnectionUtil {
+
+	private ConnectionUtil() {
+
+	}
+
+	/**
+	 * This method is used to connect database.
+	 * 
+	 * @return
+	 * @throws SQLException
+	 */
+	public static Connection getConnection() throws SQLException {
+
+		Connection connection = null;
+		String driverClass = "org.postgresql.Driver";
+		String url = "jdbc:postgresql://localhost/stadium_db";
+		String username = "postgres";
+		String password = "Siva@17104061";
+		try {
+			Class.forName(driverClass);
+			connection = DriverManager.getConnection(url, username, password);
+		} catch (Exception e) {
```

Comment:
catch(ClassNotFoundException | SQLException e){
throw new RuntimeException("Unable to connect db");
}